components/content-layer-page.tsx

```diff
@@ -34,7 +35,11 @@ export default function ContentLayerPage({
   const quote = get(pageDress, "quote");
   const title = `${primaryLabel} - 13 Vaults`;
   const sections = get(data, "sections", []);
+  const bookstore = useBookStore();
   const lead = get(pageDress, "lead");
+  const If = (props) => (props.source && isSourceEnabled(bookstore, props.source)) ||
+                        (props.version && isVVEnabled(bookstore,"version", props.version))  ?
+                          props.children : null
```

Comment:
This allows conditionally displayed elements within a vault page's markdown (see e.g. the Animal Companion page for usage examples).